nautobot/extras/tests/test_webhooks.py

```diff
@@ -100,11 +100,13 @@ class FakeResponse:
         with patch.object(Session, "send", mock_send):
             self.client.force_login(self.user)
 
-            with web_request_context(self.user, change_id=request_id):
+            # create the object to be updated in a separate context
+            with web_request_context(self.user):
                 location_type = LocationType.objects.get(name="Campus")
                 location = Location(name="Location 1", status=self.statuses[0], location_type=location_type)
                 location.save()
 
+            with web_request_context(self.user, change_id=request_id):
                 location.name = "Location Update"
```

Comment:
Should we also have tests for the "create and update in a single web_request_context" case to make sure the webhooks are correctly triggered or not triggered in that case?